Sources/StreamChat/Database/DTOs/AttachmentDTO.swift

```diff
@@ -79,6 +82,13 @@ class AttachmentDTO: NSManagedObject {
         request.predicate = NSPredicate(format: "localStateRaw == %@", LocalAttachmentState.pendingUpload.rawValue)
         return request
     }
+    
+    static func loadInProgressAttachments(context: NSManagedObjectContext) -> [AttachmentDTO] {
+        let request = NSFetchRequest<AttachmentDTO>(entityName: AttachmentDTO.entityName)
+        request.sortDescriptors = [NSSortDescriptor(keyPath: \AttachmentDTO.id, ascending: true)]
+        request.predicate = NSPredicate(format: "localStateRaw == %@", LocalAttachmentState.uploading(progress: 0).rawValue)
```

Comment:
Is there any chance we may get attachments with uploading progress different to 0 and less than 100?

Reply:
No, the rawValue is just "uploading", the progress is not accounted for the rawValue 👍